src/views/views.py

```diff
@@ -152,8 +159,18 @@ def employer_dashboard(request):
     return ProtectedViewController(request).render('pages/employer-dashboard.html')
 
 
+
+
+
 def make_job_offer(request):
-    return ProtectedViewController(request).render('pages/make_job_offer.html')
+    job_offers = JobController().get_all()
+    categories_list = CategoryController().get_categories()
+    filters = request.GET
+    if filters and filters.get('category'):
+        job_category_id = filters.get('category')
+        job_offers = Job.get_by_category(job_category_id)
```

Comment:
You used JobController up there, but now you are accessing data layer directly which is forbidden! Use JobController

Reply:
Again, identical to order by companies, why is this wrong and not for the companies?